boot/zephyr/pm.yml

```diff
@@ -21,12 +21,14 @@ mcuboot_secondary:
     after: mcuboot_primary
 
 #ifndef CONFIG_BOOT_SWAP_USING_MOVE
+#ifndef CONFIG_SINGLE_IMAGE_DFU
```

Comment:
```suggestion
#if !defined(CONFIG_BOOT_SWAP_USING_MOVE) && !defined(CONFIG_SINGLE_IMAGE_DFU)
```

Do you think that this is more readable? Just a thought.

Reply:
I don't have preference in this case. @nvlsianpu ?

Reply:
what Håkon had suggested looks more legible to me.

Reply:
changed